packages/pyolite-kernel/src/worker.ts

```diff
@@ -582,10 +462,10 @@ export class PyoliteRemoteKernel {
     reject: () => void;
     resolve: () => void;
   } | null = null;
+  protected _pyodide: Pyodide.PyodideInterface = null as any;
   /** TODO: real typing */
   protected _localPath = '';
   protected _driveName = '';
-  protected _pyodide: any;
   protected _kernel: any;
```

Comment:
These `any`s still kill me. Among the tasks we should explore in #386 are:
- doing a thorough typing of all our custom classes
- generating typescript types from the python types https://github.com/pyodide/pyodide/issues/2162